Python/ceval_gil.c

```diff
@@ -646,11 +661,9 @@ PyEval_RestoreThread(PyThreadState *tstate)
 */
 
 void
-_PyEval_SignalReceived(PyInterpreterState *interp)
+_PyEval_SignalReceived()
```

Comment:
```suggestion
_PyEval_SignalReceived(void)
```

(or take a `_PyRuntimeState *runtime` argument)